sql/select.go

```diff
@@ -49,6 +49,7 @@ func (p *planner) Select(n *parser.Select) (planNode, error) {
 	if err := scan.initTargets(n.Exprs); err != nil {
 		return nil, err
 	}
+
```

Comment:
This function grew organically which resulted in the lack of spaces, but placing a single space here seems spurious. Perhaps spaces between each of the `scan.init` sections and between group-by and order-by processing.